sdk/purview/purview-sharing-rest/swagger/README.md

```diff
@@ -0,0 +1,58 @@
+# Azure Purview Share TypeScript Protocol Layer
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+package-name: "@azure-rest/purview-sharing"
+title: Purview Sharing
+description: Purview Sharing Client
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../
+source-code-folder-path: ./src
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-02-15-preview/share.json
```

Comment:
I would recommend pinning this to a specific commit. It will save you some leg work if in the future you need to regenerate against the same swagger but main has changed.